app/victoria-metrics/test/parser.go

```diff
@@ -0,0 +1,51 @@
+// +build integration
```

Comment:
why do you need this tag here?

Reply:
we don't need that code in prod build, just in tests

Reply:
but you don't import it in prod build, only in tests which are already guarded with build tag